From dab98a0e52b9bb9930048b94d637a2fdb218fc45 Mon Sep 17 00:00:00 2001
From: Richard Jones <rjones@redhat.com>
Date: Thu, 28 Jan 2010 09:59:08 +0000
Subject: [PATCH] supermin: Add special case for libgcc_s-*.so.N

Because this file contains the GCC version and a date, it frequently
changes on the host.  Create a special case so that this file is
matched using a wildcard for the version/date part of the name.
---
 appliance/supermin-split.sh.in |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/appliance/supermin-split.sh.in b/appliance/supermin-split.sh.in
index 7b83afb..c710dc3 100755
--- a/appliance/supermin-split.sh.in
+++ b/appliance/supermin-split.sh.in
@@ -78,6 +78,10 @@ for path in $(find -not -name fakeroot.log); do
     elif [[ "$file" =~ '^libbfd-.*\.so$' ]]; then
         echo "$dir/libbfd-*.so" >&6
 
+    # Special case for libgcc_s-<gccversion>-<date>.so.N
+    elif [[ "$file" =~ '^libgcc_s-.*\.so\.([0-9]+)$' ]]; then
+        echo "$dir/libgcc_s-*.so.${BASH_REMATCH[1]}" >&6
+
     # libfoo-1.2.3.so
     elif [[ "$file" =~ '^lib(.*)-[-.0-9]+\.so$' ]]; then
         echo "$dir/lib${BASH_REMATCH[1]}-*.so" >&6
-- 
1.6.5.2

